force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
+    <isExposed>true</isExposed>
+    <masterLabel>Donation History Table</masterLabel>
+    <targets>
```

Comment:
I think we can narrow down the targets here a bit more. Is this component only intended to be hosted on pages that are related to an Account or Contact?

Reply:
Add a wrapper for this case, for now